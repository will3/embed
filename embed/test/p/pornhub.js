const test = require('../utils/test');

test('pornhub',
  'https://www.pornhub.com/view_video.php?viewkey=ph58cccc07d4dbe');
