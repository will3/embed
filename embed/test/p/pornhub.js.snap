// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` pornhub 1`] = `
Object {
  "description": undefined,
  "originalUrl": "https://www.pornhub.com/view_video.php?viewkey=ph58cccc07d4dbe",
  "site_name": "pornhub",
  "thumbnail": Object {
    "url": "https://bi.phncdn.com/videos/201703/18/110022832/original/(m=eGcE8daaaa)12.jpg",
  },
  "title": "Big Titty Girlfriend Fucked Rough And Creampied - Pornhub.com",
  "video": Object {
    "description": undefined,
    "height": "360",
    "title": undefined,
    "url": "https://www.pornhub.com/embed/ph58cccc07d4dbe",
    "width": "640",
  },
}
`;
